CHANGELOG.rst

```diff
@@ -55,9 +54,9 @@ Changed
 - removed ``--pre_load`` from run command (Rasa NLU server will just have a maximum of one model and that model will be
   loaded by default)
 - changed file format of a stored trained model from the Rasa NLU server to ``tar.gz``
-- ``rasa train`` uses fallback config if an invalid config is given
-- ``rasa test core`` compares multiple models if a list of model files is provided for the argument ``--model``
-- ``rasa train`` falls back to ``rasa train core``/``rasa train nlu`` if the corresponding training data are missing
+- train command uses fallback config if an invalid config is given
+- test command now compares multiple models if a list of model files is provided for the argument ``--model``
+- train command falls back to training only nlu or core if the corresponding training data are missing
```

Comment:
This is default behaviour, so you don't need to add it in - previously, combined training was not possible.